Client/Assets/LoginsHelper.js

```diff
@@ -599,19 +599,28 @@ function onBlur(event) {
   LoginManagerContent.onUsernameInput(event)
 }
 
-window.addEventListener("load", function(event) {
+var documentBody = document.querySelector('body')
+var observer = new MutationObserver(function(mutations) {
+    findLogins()
+});
+
+observer.observe(documentBody, { attributes: false, childList: true, characterData: false });
```

Comment:
I wonder if you also want to include `subtree: true` here? From the docs, it looks like `childList` might just be direct descendants. Worth testing, though (speaking of which, it might be a good idea to add a simple test case for dynamic `form` insertions).